l10n_ch_states/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
Your editor is not putting a EOL before EOF. Please fix this to make travis happy.